test/service/test_manifest.py

```diff
@@ -82,6 +86,15 @@ def _get_manifest_url(self, format_, filters):
         service = ManifestService(StorageService())
         return service.get_manifest(format_, filters)
 
+
+class TestManifestEndpoints(ManifestTestCase):
+
+    def run(self, result: Optional[unittest.result.TestResult] = ...) -> Optional[unittest.result.TestResult]:
+        # Suppress caching of manifests to prevent false assertion positives
+        with mock.patch('azul.service.manifest_service.ManifestService._can_use_cached_manifest') as m:
+            m.return_value = False
+        return super().run(result)
```

Comment:
Still wrong. I think you're confused about the evaluation order of return statements in context managers but that's essential Python knowledge that you need to understand. Of course, this could just be an oversight, I don't know.

Anyways, now I have my doubts if this is even effective. I would like you leave this broken as is and add an assertion to one of the tests that this should—but currently doesn't—patch. The assertion should fail if a cached manifest is reused in the test. I need to see the Travis build fail on that assertion. Then we can fix problem.

Reply:
@hannes-ucsc yes this is completely a mishap on my behave. 
For your doubts, if I may remind you, we added `Cache single and multipart manifests separately` meaning we use single part upload boolean as a parameter to determine the object key. Being the reason why this test passed with out having to use the deactivated use cached manifest. 
I'll still add the assertion.

Reply:
@achave11 You may be right but, you see, only if you didn't make any additional mistakes. Since you made one mistake it was justified for me to assume that there were others. And that's why I had doubts and that's why I asked for the added tests.